package.json

```diff
@@ -0,0 +1,115 @@
+{
+  "name": "network-dashboard",
+  "description": "Aragon Network Dashboard",
+  "version": "0.1.0",
+  "private": true,
+  "license": "AGPL-3.0-or-later",
+  "author": "Aragon Association <legal@aragon.org>",
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/aragon/network-dashboard.git"
+  },
+  "bugs": {
+    "url": "https://github.com/aragon/network-dashboard/issues"
+  },
+  "homepage": "https://github.com/aragon/network-dashboard#readme",
+  "keywords": [
+    "aragon",
+    "network",
+    "ethereum",
+    "dapp",
+    "blockchain",
+    "startups"
+  ],
+  "engines": {
+    "node": ">= 10.0.0"
+  },
+  "bundlewatch": {
+    "files": [
+      {
+        "path": "public/*.js",
+        "maxSize": "1600kb",
+        "compression": "gzip"
+      }
+    ]
+  },
+  "dependencies": {
+    "@aragon/ui": "^1.5.0",
+    "@sentry/browser": "^5.17.0",
+    "core-js": "^3.3.2",
+    "dotenv": "^8.2.0",
+    "prop-types": "^15.6.2",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-spring": "^7.2.10",
+    "regenerator-runtime": "^0.13.3",
```

Comment:
We shouldn’t need this if we target modern browsers (with `async` / `await` support).